conan/tools/cmake/cmake.py

```diff
@@ -147,7 +147,9 @@ def install(self, build_type=None, component=None, strip=False):
         arg_list = ["--install", build_folder, build_config, "--prefix", pkg_folder]
         if component:
             arg_list.extend(["--component", component])
-        if strip:
+
+        do_strip = self._conanfile.conf.get("tools.cmake.install:strip", check_type=bool)
```

Comment:
```suggestion
        do_strip = self._conanfile.conf.get("tools.cmake:install_strip", check_type=bool)
```
It would be a bit more aligned with existing configs ``tools.cmake:make_program``